src/makeCalculator.js

```diff
@@ -37,7 +37,34 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calculator = {
```

Comment:
you can return immediately

```suggestion
  return {
```